staging/src/k8s.io/component-base/featuregate/feature_gate.go

```diff
@@ -218,7 +218,8 @@ func (f *featureGate) SetFromMap(m map[string]bool) error {
 		k := Feature(k)
 		featureSpec, ok := known[k]
 		if !ok {
-			return fmt.Errorf("unrecognized feature gate: %s", k)
+			klog.Warningf("unrecognized feature gate: %s", k)
```

Comment:
why do we need this commit? Do we know who added it? We should really get used to document that in the commit messages. With the rebase procedure we lose this information otherwise.

Reply:
More than better commit messages, I think there needs to be a renewed emphasis on commenting in the code.  If/when we squash the commit message may lose relevancy, but the comments stay.